usaspending_api/awards/serializers_v2/serializers.py

```diff
@@ -6,11 +6,11 @@ class AwardTypeAwardSpendingSerializer(serializers.Serializer):
     obligated_amount = serializers.DecimalField(None, 2)
```

Comment:
Can you restructure the directories in the `awards` django app to match what `references` is doing with v1 and v2?